core-java/src/main/java/com/baeldung/ExceptionHandling/Exceptions.java

```diff
@@ -0,0 +1,251 @@
+	    try (Scanner contents = new Scanner(new File(playerFile))) {
+	      return Integer.parseInt(contents.nextLine());
+	    } catch (FileNotFoundException e ) {
+	      logger.warning("File not found, resetting score.");
+	      return 0;
+	    }
+	}
+	
+	
+	public int getPlayerScore7(String playerFile) {
+	    try (Scanner contents = new Scanner(new File(playerFile))) {
+	        return Integer.parseInt(contents.nextLine());
+	    } catch (IOException e) {
+	        logger.warning("Player file wouldn't load!");
+	        return 0;
+	    } catch (NumberFormatException e) {
+	        logger.warning("Player file was corrupted!");
+	        return 0;
+	    }
+	}
+	
+	
+	public int getPlayerScore8(String playerFile) {
+	    try (Scanner contents = new Scanner(new File(playerFile)) ) {
+	        return Integer.parseInt(contents.nextLine());
+	    } catch (FileNotFoundException e) {
+	        logger.warning("Player file not found!");
+	        return 0;
+	    } catch (IOException e) {
+	        logger.warning("Player file wouldn't load!");
+	        return 0;
+	    } catch (NumberFormatException e) {
+	        logger.warning("Player file was corrupted!");
+	        return 0;
+	    }
+	}
+		
+	
+	public int getPlayerScore9(String playerFile) {
+	    try (Scanner contents = new Scanner(new File(playerFile))) {
+	        return Integer.parseInt(contents.nextLine());
+	    } catch (IOException | NumberFormatException e) {
+	        logger.warning("Failed to load score!");
+	        return 0;
+	    }
+	}
+	
+	
+	public List<Player> loadAllPlayers2(String playersFile) throws TimeoutException {
+		boolean tooLong = true;
+		
+	    while (!tooLong) {
+	        // ... potentially long operation
+	    }
+	    throw new TimeoutException("This operation took too long");
+	}
+	
+	
+	public List<Player> loadAllPlayers3(String playersFile) throws TimeoutException {
+	    if(!isFilenameValid(playersFile)) {
+	        throw new IllegalArgumentException("Filename isn't valid!");
+	    }
+		return null;
+	   
+	    // ...
+	}
+	
+	public List<Player> loadAllPlayers4(String playersFile) 
+			  throws IOException {
+			    try { 
+			       throw new IOException();
+			    } catch (IOException io) { 		
+			        throw io;
+			    }
+			}
+	
+	
+	
+	public List<Player> loadAllPlayers5(String playersFile) 
+			  throws PlayerLoadException {
+			    try { 
+			        throw new IOException();
+			    } catch (IOException io) { 		
+			        throw new PlayerLoadException(io);
+			    }
+			}
+	
+	
+	public List<Player> loadAllPlayers6(String playersFile) {
+	    try {
+	        throw new NullPointerException();
+	    } catch ( Throwable t ) {
+	        throw t;
+	    }
+	}
+	
+	 class FewerExceptions extends Exceptions {	
+	    @Override
+	    public List<Player> loadAllPlayers(String playersFile) {  //can't add "throws MyCheckedException 
+			return null;
+	        // overridden
+	    }
+	}
+	 
+	 public void doSomething() {

```

Comment:
Let's make sure that the method names for the anti-patterns have something like "antiPattern" in the name. Make it clear that the users should **not** do this.